dcpy/lifecycle/distribute/socrata.py

```diff
@@ -53,27 +62,31 @@ def dist_from_local(
             metadata=md,
             dataset_destination_id=dataset_destination_id,
             dataset_package_path=package_path,
-            publish=publish,
-            metadata_only=metadata_only,
+            publish=bool(publish),
+            metadata_only=bool(metadata_only),
         )
     except Exception as e:
         return f"Error pushing {md.attributes.display_name}, destination: {dest.id}: {str(e)}"
 
 
 def dist_from_local_all_socrata(
     package_path: Path,
-    **pub_kwargs,
+    **pub_kwargs: Unpack[PublishKwargs],
```

Comment:
Ooh this is fun

Reply:
So do the required ones throw a runtime error if missing?

Reply:
I thought you'd like this. And yes, it behaves just as if they were literally written out kwargs.